client/src/reducers/controls.js

```diff
@@ -2,7 +2,7 @@
 
 import _ from "lodash";
 
-import { World, WorldUtil } from "../util/stateManager";
+import { World, WorldUtil, ControlsHelper } from "../util/stateManager";
```

Comment:
IMO the code would be easier to read if you imported the helper functions directly from the ControlsHelper files:

`import {createCategoricalSelectionState} from "../util/stateManager/ControlsHelper";`

```
# old 
ControlsHelper.createCategoricalSelectionState()
# new
createCategoricalSelectionState()
```

Reply:
This is another case I find the opposite to be true, as I lose the connection to where the code is defined.    But I'm super happy to follow a standard if one is established.

Do you want to talk to Colin about codifying this stuff into some standard?   I generally think consistency trumps any given position :-).     I'll happily fix my code.